manageability-docker/TBH/Release-README

```diff
@@ -0,0 +1,24 @@
+QUICK README
+=============
+1.  To install both TC and BC run the install-inb.sh script with root privileges. Before running the script update the following fields in azure_conf_file with the necessary information to have device connected to cloud.
```

Comment:
We need to remove references to TC and BC.  Just use INBM or Intel In-band Manageability instead.